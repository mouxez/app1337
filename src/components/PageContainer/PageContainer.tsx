import { FC, ReactNode } from "react";
import styles from "./PageContainer.module.css";

const { container } = styles;

type PageContainerProps = {
  children: ReactNode;
};

const PageContainer: FC<PageContainerProps> = ({ children }) => {
  return <div className={container}>{children}</div>;
};

export default PageContainer;
