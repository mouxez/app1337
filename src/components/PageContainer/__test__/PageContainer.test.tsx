import { render } from "@testing-library/react";
import PageContainer from "../PageContainer";

describe("components/PageContainer", () => {
  it("renders correctly", async () => {
    const { asFragment } = render(
      <PageContainer>
        <span>I am a child!</span>
      </PageContainer>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
