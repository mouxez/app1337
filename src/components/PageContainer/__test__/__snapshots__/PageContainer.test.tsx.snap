// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/PageContainer renders correctly 1`] = `
<DocumentFragment>
  <div
    class="container"
  >
    <span>
      I am a child!
    </span>
  </div>
</DocumentFragment>
`;
